docs/deployments/ASF-deployment.md

```diff
@@ -78,5 +93,6 @@ aws cloudformation deploy \
         EDLUsername=<earthdata-login-username> \
```

Comment:
Should we remove `EDLUsername` and `EDLPassword`?

Reply:
Ah rats, I thought I did! Fixed.